src/concepts/analysis_gotchas.md

```diff
@@ -244,3 +244,15 @@ Summary of reasons for this decision:
 In general, data coming from an application instance not run by a human is not wanted in analysis. As of this writing, [GeckoDriver](https://github.com/mozilla/geckodriver) (one of the official mechanisms to launch and control an automated version of Firefox for e.g. web compatibility testing) is [configured _not_ to send Telemetry by default](https://searchfox.org/mozilla-central/rev/baf1cd492406a9ac31d9ccb7a51c924c7fbb151f/testing/geckodriver/src/prefs.rs#154) but we can't control for other things people might do in the field.
 
 On desktop, one field to watch out for is headless mode (`environment.system.gfx.headless` in the main ping): if that field is set, you are for certain not working with a version of Firefox being operated by a real human. You can see an example of some client pings with this field set skewing the nightly numbers in [bug 1643341](https://bugzilla.mozilla.org/show_bug.cgi?id=1643341). An easy solution is to just filter out these types of clients in your analysis. You can see an example of this pattern in [this query](https://sql.telemetry.mozilla.org/queries/71781/source).
+
+## Build Ids
+
+Generally speaking, application versions are monotonically increasing multipart alphanumeric strings like "89.0a1" or "68.0.3".
+Build Ids are not this.
+A Build Id is a sequence of characters that identify the specific code and configuration built into a product.
+It helps identify which code changes were included in a build as well as what features may have been enabled at build time.
+For example, in Firefox Desktop, the build id is the date and time the build was built in yyyymmddhhmmss format.
+A build id might be formatted in any way and contain the time or version control system revision of the code included in the build.
+
+Do not assume the build id format does not vary between products, between channels of the same product, or over time within the same channel of the same product.
+(The build id format for Firefox Desktop has been very stable over time thus far, but even it can be different for different platforms in some respin circumstances (if e.g. only one platform's builder failed)).
```

Comment:
"Not not" tends to be a bit confusing, especially for non-english-native speakers. 

```suggestion
Do not assume build id's are consistent across the products we ship. A build id format may vary between products, between channels of the same product, or over time within the same channel of the same product.
The build id format for Firefox Desktop has been very stable over time thus far, but even it can be different for different platforms in some respin circumstances (if e.g. only one platform's builder failed).
```